python/modules/IcePy/Init.cpp

```diff
@@ -122,31 +122,48 @@ logger : Ice.Logger
 
     constexpr const char* IcePy_loadSlice_doc = R"(loadSlice(args: list[str]) -> None
 
-Dynamically loads Slice definitions from one or more files and generates the corresponding Python code.
+Compiles Slice definitions and loads the generated code directly into the current Python environment.
+
+This function does not generate any Python source files. Instead, the generated Python code is loaded directly into the
+running interpreter.
+
+This function does not generate any code for Slice files included by the Slice files being loaded. It is the caller's
+responsibility to load all necessary Slice definitions. This can be done in a single call to :func:`loadSlice` by
+providing all Slice files (including included files) in the `args` parameter, or by making multiple calls to
+:func:`loadSlice`.
+
+When :func:`loadSlice` is called multiple times, the definitions that are already loaded are reused and only new
```

Comment:
What about:
```
When :func:`loadSlice` is called multiple times with the same Slice file, the corresponding Python code is not regenerated nor reloaded. 
```

and overlook the situation where you add new definitions to that Slice file between reload ... ultra unusual.